src/tokens/core.js

```diff
@@ -67,16 +66,29 @@ export default {
     Semibold: 600,
     Bold: 700,
   },
-  LineHeight: {
```

Comment:
can we add this to MediaQuery too, or do I need to just add the `px` in my app? This is how I'm currently using media queries. ```[`@media(max-width: ${theme.MediaQuery.Med})`]: {
      margin: 0,
      paddingTop: '52px',
    },```

Reply:
I'm torn on this one. I was about to add it but then I thought about the majority of use cases in raja are in javascript and number comparisons. I'm almost leaning towards duplicating this information into another object with a different name. The name MediaQuery to me implies CSS.

Reply:
@jgerlach Updated, let me know your thoughts.